### PR TITLE
Fix some typos in data-registries.md and import.md

### PR DESCRIPTION
**Pull request by ZhengRui**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/import.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ outs:
 ```
 
 Several of the values above are pulled from the original `.dvc` file
-`model.pkl.dvc` in the external DVC repository. The `url` and `rev_lock`
+`data.xml.dvc` in the external DVC repository. The `url` and `rev_lock`
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import.md:158]: Thanks, that wasn't right!

But I think it'll be even less confusing to do this:

```suggestion
 [in the external DVC repository](https://github.com/iterative/example-get-started/blob/master/data/data.xml.dvc). The `url` and `rev_lock`
```

File: content/docs/use-cases/data-registries.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ $ dvc import https://github.com/example/registry images/faces
 ```
 
 Besides downloading the data, importing saves the information about the
-dependency from the local project to the data source (registry repo). This is
+dependency from the data source (registry repo) to the local project. This is
```
Comment (jorgeorpinel) [inline content/docs/use-cases/data-registries.md:123]: Hmm no, I realize this wasn't worded clearly, but we mean the dependency that the local project has to the data source i.e. the local project depends on the data source. Let me think... ⌛

Comment (jorgeorpinel) [inline content/docs/use-cases/data-registries.md:123]: Let's do this for now:
```suggestion
dependency that the local project has on the data source (registry repo). This is
```

**Comment by jorgeorpinel** [comment]
Thank you @ZhengRui! This helps us find areas to improve the explanations in our docs.

**Comment by jorgeorpinel** [comment]
Merged the restyled PR which merged this automatically, but I'll see if there's an even better way to clarify this in the Data Registry use case.

File: content/docs/use-cases/data-registries.md (hunk 2 above)
Comment (ZhengRui) [reply content/docs/use-cases/data-registries.md]: I see what you mean, maybe "dependency of the local project on the data source (registry repo)"

**Pull request merged**